src/igvfd/schemas/model.json

```diff
@@ -62,6 +62,12 @@
         "schema_version": {
             "default": "3"
         },
+        "award": {
```

Comment:
add comment explaining the restriction, on the way to deprecate use of the schemas

Reply:
Added.